### DISCUSSION TITLE
West Coast Crabber/Towboat Lane

### DISCUSSION THREAD

**Discussion by xavierlh1** [jaybo/deepzoom-blog#17] [Ideas]
Crabbers and Tugboat operators in Washington, Oregon and California have a voluntary agreement for Crabber/Towboat Lane (details at https://wsg.washington.edu/community-outreach/outreach-detail-pages/crabbertowboat-lane-agreements-download-charts-data-and-meetings/) and publish a map of these (https://www.google.com/maps/d/viewer?mid=11Aq9QtgPf3L_qYp7emomO-CxB1Q&ll=46.063893315481245%2C-123.52230611857915&z=9) unfortunately over a Google map which is not anywhere as useful to mariners as a proper navigation chart.

I thought of entering the ends of these lanes as markers in DeppZoom, but I'm hoping that you may have a better method of overlaying this information.

**Comment by jaybo** [comment]
Hi.

Unfortunately, DeepZoom considers all GeoJSON geometry to be either a route or a marker.  In this case you have GeoJSON (or KML) lines and polygons which are just borders.  When importing this data, DeepZoom converts everything to a route, resulting in the hilarity of  having 68 routes simultaneously active.  It works for static display purposes, but as soon as you try and add a "real" route to the mix, things become trying.

The best eventual solution would be to have a checkbox when importing (and even when creating geometry in DeepZoom) which would convey the meaning of "treat this polyline as just a line, but not a route".

I used DeepZoom3 which is coming out in a day or so to create this route.  
https://deepzoom.com/trip/vbbzth7e

The sequence to create the trip was:

1. In Google maps, "...", Download KML, Check "Export as KML instead of KMZ.
2. In DeepZoom, Trips.Import the file you downloaded. (Hint: just drag and drop the KML file on anywhere on the right column)

You may want to hold off doing much with this trip until DeepZoom3 is officially released since a few things aren't backward compatible with V2.  In particular, you need to manually set the chart opacity until V3 is released.

![image](https://user-images.githubusercontent.com/1607323/221999836-1e4ef7cc-72e1-4b55-a23b-06f503a2aac4.png)

**Comment by xavierlh1** [comment]
Wow!  Although this is impractical with the current feature set as you point out, I'm quite impressed with what you were able to do.  I'm a new user exploring the tool in preparation for a sailing trip up the coast from Oregon, through the San Juans, Georgia Straits and north from there. I'm now eagerly waiting V3!
